src/components/dashboard/FaceVerification/screens/IntroScreen.js

```diff
@@ -155,6 +159,50 @@ const IntroScreen = ({ styles, screenProps }) => {
       </Section>
     </Wrapper>
   )
+
+  const introScreenB = (
+    <Wrapper>
+      <Section style={styles.topContainer} grow>
+        <View style={styles.mainContentB}>
+          <Section.Title fontWeight="bold" textTransform="none" style={styles.mainTitleB}>
+            {`${getFirstWord(fullName)},`}
+            <Section.Text fontWeight="normal" textTransform="none" fontSize={24} lineHeight={30}>
+              {'\nVerify you are a real\nlive person'}
+            </Section.Text>
+          </Section.Title>
+          <Section.Text fontSize={18} lineHeight={25} letterSpacing={0.18} style={{ marginTop: '17px' }}>
+            Your image is only used to prevent the creation of duplicate accounts and will never be transferred to any
+            third party
+          </Section.Text>
+          {/* TODO: where should learn more point to? */}
+          <Section.Text
+            fontWeight="bold"
+            fontSize={18}
+            lineHeight={26}
+            textDecorationLine="underline"
+            style={styles.learnMore}
+            onPress={openPrivacy}
+          >
+            Learn More
+          </Section.Text>
+          <View style={styles.illustrationB}>
+            <FashionShootSVG />
+          </View>
+          <CustomButton
+            style={[styles.button]}
+            onPress={handleVerifyClick}
+            disabled={false !== disposing}
+            contentStyle={{ minHeight: '53px' }}
+            textStyle={{ fontSize: '16px', lineHeight: '19px', letterSpacing: '0.49px', fontWeight: 'bold' }}
+          >
+            OK, VERIFY ME
+          </CustomButton>
+        </View>
+      </Section>
+    </Wrapper>
+  )
+
+  return AB === 'A' ? introScreenA : introScreenB
```

Comment:
intro screen doesnt fit inside iphone 5s screen.

Reply:
the original screen never seem to have fit the 5s screen to begin with

Reply:
done

Reply:
You actually rendering 2 large jsx trees, then returning one of them.
To optimize performance in the such cases React recommends to use composition.

Please, do the same way like with the `Instructions` component: 

- move those introScreenA/B to the separate components (here or inside `components` folder) 
- select which component you need to render (via `useMemo`)
- pass all the props needed

Reply:
i wouldnt call it large. this is an overkill for a temporary ab testiing

Reply:
@sirpy anyway it's a breaking of React rules. to make them separate components won't take a lot of time
@sirpy @omerzam  if you want i could do it myself

Reply:
if you can do it quickly then go ahead

Reply:
ok will look a bit later